scripts/morty/morty_pde_ode_compare.py

```diff
@@ -0,0 +1,413 @@
+                if not vector_form:
+                    return_list.append(term2)
+                else:
+                    return_list.append(term2[zi])
+            else:
+                if not vector_form:
+                    return_list.append(0.5 * (term1 + term2))
+                else:
+                    return_list.append(0.5 * (term1[zi] + term2[zi]))
+        return np.asarray(return_list)
+
+
+class DiffEqComp(FormatAssist):
+    def __init__(self, mu, S, isotope, nu1, nu2, z1, z2,
+                 nodes, tf, dt, initial_guess, lmbda):
+        """
+        This class generates results for comparing differential equations.
+        Particularly, this class compares ODE depletion in time to
+        PDE depletion in 1D space and time.
+
+        Parameters
+        ----------
+        mu : dict
+            key : string
+                Name of isotope
+            value : List of float
+                Value of loss in-core and value ex-core
+        S : dict
+            key : string
+                Name of isotope
+            value : List of float
+                Value of source in-core and value ex-core
+        isotope : string
+            Name of isotope to model
+        nu1 : float
+            Linear flow rate in-core
+        nu2 : float
+            Linear flow rate ex-core
+        z1 : float
+            Linear height of in-core region
+        z2 : float
+            Linear height of ex-core region
+        nodes : int
+            Number of spatial nodes (split between in- and ex-core)
+        tf : float
+            End time value in seconds
+        dt : float
+            Time step value in seconds
+        initial_guess : List of floats
+            Initial concentration in-core and ex-core
+        lmbda : float
+            Numerical stability value (must be <1);
+            Time step times flow rate divided by spatial step
+
+        """
+        self.mu1, self.mu2 = mu[isotope]
+        self.S1, self.S2 = S[isotope]
+        self.nu1, self.nu2 = nu1, nu2
+        self.nodes = nodes
+        self.zs = np.linspace(0, z1+z2, nodes)
+        self.dt = dt
+        self.tf = tf
+        self.ts = np.arange(0, tf+dt, dt)
+        self.z1 = z1
+        self.init_conc = self._format_spatial(initial_guess[0],
+                                              initial_guess[1])
+        self.lmbda = lmbda
+        return
+
+    def _format_spatial(self, term1, term2, vector_form=False):
+        """
+        Distribute term 1 in < z1, and term 2 in > z1.
+        Returns a list of terms corresponding to each z.
+        If term1 and term2 are vectors, use `vector_form=True`
+
+        Parameters
+        ----------
+        term1 : float
+            Term in the in-core region
+        term2 : float
+            Term in the ex-core region
+        vector_form : bool
+            Terms are given as spatial vectors rather than floats
+
+        Returns
+        -------
+        return_list : 1D numpy array
```

Comment:
```suggestion
        return_list : :class:`np.ndarray`
```

For future better formatting.